charts/theia.cloud/values.yaml

```
@@ -161,14 +161,14 @@ keycloak:
 # @default -- (see details below)
 operator:
   # -- The operator image
-  image: theiacloud/theia-cloud-operator:0.8.1.MS2v2
+  image: theiacloud/theia-cloud-operator:0.8.1.MS3v2
```

As we are  increasing the milestone number,  the `v2` should be removed
```suggestion
  image: theiacloud/theia-cloud-operator:0.8.1.MS3
```